src/Services/Persistence/MySql.php

```diff
@@ -72,6 +72,29 @@ public static function getOne(string $tableName, array $where)
 
     /**
      * @param string $tableName
+     * @param array  $where
+     *
+     * @return array
+     * @throws Exceptions\ConnectionException
+     * @throws Exceptions\QueryException
+     */
+    public static function getMany(string $tableName, array $where = [])
+    {
+        $sql = 'select * from `' . $tableName . '`';
+        $clauses = [];
+
+        foreach ($where as $columnName => $columnValue) {
+            $clauses[] = '`' . $columnName . '` = "' . self::getConnection()->escape_string($columnValue) . '"';
+        }
+
+        $sql   .= count($clauses) > 0 ? ' where ' . implode(' AND ', $clauses) : '';
+        $result = mysqli_fetch_all(self::query($sql), MYSQLI_ASSOC);
+
+        return $result ?: [];
+    }
```

Comment:
The most common use case when you want to get multiple results is that you are actually searching for something and it might not be there, take /hunts/active GET for example, there might be no active hunts because no one started one but there may be multiple.

In the case of getOne, you usually search by a primary key, since you know the primary key, that means that you already got it from somewhere at a certain point and it should still be in the db